src/core/github-api.js

```diff
@@ -21,8 +21,11 @@ export function githubRequestHeaders(conf) {
 }
 
 export async function getRateLimit(conf) {
+  const githubRateLimitAPI = conf.githubRateLimitAPI
+    ? conf.githubRateLimitAPI
```

Comment:
Please use `conf.githubAPI` instead. The URL should always be:

```JS
new URL("./rate_limit", githubAPI).href
```

So you will need to rename `ratelimit.json` to `rate_limit`.

Reply:
The githubAPI currently defaults to https://api.github.com/repos/${org}/${repo}
I will make convert it to https://api.github.com/repos/ and make the required changes in all other files.

Reply:
Ok, as long as it works that same, then go for it.

Reply:
@devanshbatra04 ... could you check something for me? when you make a `Request` to Github API, I think the Response's headers actually say how many remaining requests there are (`X-Rate-Limit:` or something)... if that's true, maybe we don't need getRateLimit() at all?

Reply:
Thats right. I wanted to take that path earlier. There's a blocker though. I believe we would need the number of requests before we make the multiple requests required. The rate limit api doesnt count to a decrement in requests.

Reply:
Ok, let's fix this first, as it's blocking us from merging other things as travis keeps falling over. Then we can work out a better strategy...